### ISSUE TITLE
SUL logo treatment

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#339]
1. Alt text for SUL logo should be:  ""  (currently "Sul logo").
2. Add the abbreviated logo for xs width.
3. Add explicit "sr-only" link text.

```
 <a href="http://library.stanford.edu">
       <span class="sr-only">Stanford University Libraries</span>
      <img alt="" class="hidden-xs" src="SUL-Logo-white-text-h25.png">
      <img alt="" class="visible-xs" src="SUL-Logo-Small-white-text-h25.png">
      </a>
```

Logo source: https://consul.stanford.edu/display/Logos/Home
